packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -346,9 +346,10 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
     this.notifyStateChange((draft: GraphChatListClient) => {
       // find the chat thread
       const chatThreadIndex = draft.chatThreads.findIndex(c => c.id === event.message.chatId);
-      const chatThread = draft.chatThreads[chatThreadIndex];
+      const chatThread: GraphChatThread | undefined = draft.chatThreads[chatThreadIndex];
```

Comment:
I actually had this fix in my branch - but I guess both achieve the same result. Just sharing to see if you see any pro/cons or neutral.

`      const chatThread = chatThreadIndex > -1 ? draft.chatThreads[chatThreadIndex] : undefined;

      if (
        chatThread &&
`

Reply:
Doesn't really matter, they do the same thing, but draft.chatThreads[chatThreadIndex] will be undefined whenever the chatThreadIndex was -1, so it isn't necessary to be explicit. But I don't have a problem with that either.